### ISSUE TITLE
Failed to execute 'setAttribute' on 'Element': 'value$' is not a valid attribute name.

### ISSUE THREAD

**Issue by liuwenchao** [Polymer/polymer#2666]
```
Uncaught InvalidCharacterError: Failed to execute 'setAttribute' on 'Element': 'value$' is not a valid attribute name.
```

Code:
https://github.com/liuwenchao/aha-table/blob/master/src/aha-table.html#L280

The error will disappear if I remove L280, L286. L292, L298. Any idea?

Refer to https://github.com/liuwenchao/aha-table/issues/25

**Comment by jmcgdz** [comment]
:+1:

**Comment by azakus** [comment]
Ah this looks like some IE workaround that actually breaks in Chrome.

@kevinpschaaf small example: http://jsbin.com/xugayi/edit?html,console,output

The culprit is https://github.com/Polymer/polymer/blob/master/src/lib/annotations/annotations.html#L316. Maybe we should check that the browser is IE, or through a dumb try/catch around it.

**Comment by JeremybellEU** [comment]
@azakus #2673 should contain the fix for this issue

**Comment by sky-dev** [comment]
:+1:
This bug breaks all of our input components.  It would be great if this fix could be included in Polymer 1.2.3.  

Thanks for figuring this one out @JeremybellEU.  We've been able to patch the issue using your fix for now.
